### PR TITLE
Fix minor markdown and text issues for Discovery

### PR DESCRIPTION
**Pull request by n3wscott**
Fixed a couple typos and markdown lint issues as I was reading the doc again.

Signed-off-by: Scott Nichols <snichols@vmware.com>

### CODE REVIEW

**Comment by n3wscott** [comment]
@duglin the url checker is broken:

```
./community/SDK-maintainer-guidelines.md: Can't load url: https://www.conventionalcommits.org/en/v1.0.0/ (tried 5 times)
> ./community/SDK-PR-guidelines.md
./community/SDK-PR-guidelines.md: Can't load url: https://www.conventionalcommits.org/en/v1.0.0/#summary (tried 5 times)
```

File: community/SDK-PR-guidelines.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ git checkout -b 48-fix-http-agent-error
 ## Commit Messages
 
 Please follow the
-[Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary).
+[Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary).
```
Comment (duglin) [inline community/SDK-PR-guidelines.md:45]: @lance you ok with this?

**Comment by duglin** [comment]
LGTM
if @lance is ok with the conventionalcommits version change

**Comment by lance** [comment]
@duglin @n3wscott it's not clear to me why we would point to beta-4 when 1.0.0 is already released.

**Comment by n3wscott** [comment]
> @duglin @n3wscott it's not clear to me why we would point to beta-4 when 1.0.0 is already released.

ah, because they had a blip, and stopped serving 1.0.0 for a moment. 

Let me take that part out. It was failing the CI at 1.0.0 because https://www.conventionalcommits.org/en/v1.0.0/#summary was a dead link 2 days ago

**Comment by duglin** [comment]
Approved on the 7/30 call

**Pull request merged**